packages/position/test/index.js

```diff
@@ -1,61 +1,52 @@
-import 'jest-styled-components'
 import React from 'react'
 import TestRenderer from 'react-test-renderer'
-import {
-  Position,
-  Relative,
-  Absolute,
-  Fixed,
-  Sticky
-} from '../src'
+import { Position, Relative, Absolute, Fixed, Sticky } from '../src'
 
 const renderJSON = el => TestRenderer.create(el).toJSON()
 
-describe('@rebass/position', () => {
-  test('Position renders', () => {
-    const json = renderJSON(
-      <Position
-        position='fixed'
-        top='0'
-        right='0'
-        bottom='0'
-        left='0'
-        zIndex='1'
-      />
-    )
-    expect(json).toHaveStyleRule('position', 'fixed')
-    expect(json).toHaveStyleRule('z-index', '1')
-    expect(json).toHaveStyleRule('top', '0')
-    expect(json).toHaveStyleRule('right', '0')
-    expect(json).toHaveStyleRule('bottom', '0')
-    expect(json).toHaveStyleRule('left', '0')
-  })
+test('Position renders', () => {
```

Comment:
Did you mean to remove the `describe`?

Reply:
I think this is cleaner considering that the file is already scope. `describe` helps with deeper scope like in root tests.

Reply:
Sure but this change is unrelated to the PR